### DISCUSSION TITLE
Add Nishimura to Stellarium

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3374] [General]
Nishimura (C/2023 P1) has orbital elements mentioned on the page. Is it possible to add it to Stellarium?

http://www.aerith.net/comet/catalog/2023P1/2023P1.html

**Comment by gzotti** [comment]
As with any other, use the SolarSystemEditor plugin.

**Comment by alonsoac** [comment]
is there any way to see this comet in Stellarium web?

**Reply by gzotti** [reply]
wrong forum. https://github.com/Stellarium/stellarium-web-engine
